community/networkmanager/networkmanager.pre-install

```diff
@@ -2,6 +2,8 @@
 
 addgroup -S plugdev 2>/dev/null
 
-printf "  *\n  * To setup system connections, regular users must either use Polkit for authentication or be a member of the 'plugdev' group."
+printf "  *\n  * To setup system connections, regular users must be member of 'plugdev' group.\n  *\n"
+printf "  *\n  * To control WiFi devices, enable wpa_supplicant service: 'rc-update add wpa_supplicant default'\n"
+printf "  * then reboot the system or restart 'wpa_supplicant' and 'networkmanager' services respectively.\n  *\n"
 
```

Comment:
Hm, can't wpa_supplicant be dbus-activated from NM too? Then users wouldn't have to do this and we wouldn't have to mess with the initd service

Reply:
I have no idea if openrc can dbus-activate services, I know that's how it's done on systemd systems.

Reply:
Ah no, I was taking about NM starting wpa_supplicant directly. But I guess activating the service isn't too bad.